deploy/addons/storage-provisioner/storage-provisioner.template

```diff
@@ -51,7 +51,7 @@ spec:
   hostNetwork: true
   containers:
   - name: storage-provisioner
-    image: {{default "gcr.io/k8s-minikube" .ImageRepository}}/storage-provisioner-ARCHTAG_NONE:v1.8.1
+    image: {{default "gcr.io/k8s-minikube" .ImageRepository}}/storage-provisioner-{{default "ARCHTAG_NONE" .ArchTag}}:v1.8.1
```

Comment:
ARCHTAG_NONE feels like a hack, but I don't honestly understand it enough. 

I don't want to violate Chesterton's Fence (https://en.wikipedia.org/wiki/Wikipedia:Chesterton%27s_fence), but I wonder if this template macro just be safely removed with this new model, in preference to `storage-provisioner:v1.1.1`?

I believe all of the mentions of ARCHTAG in these template files should be replaced with {{ .Arch }}, however.

Reply:
fair enough